domain/grant.go

```diff
@@ -13,24 +13,24 @@ const (
 )
 
 type Grant struct {
-	ID             string
-	Status         GrantStatus
-	AccountID      string
-	AccountType    string
-	ResourceID     string
-	Role           string
-	Permissions    []string
-	ExpirationDate *time.Time
-	AppealID       string
-	RevokedBy      string
-	RevokedAt      *time.Time
-	RevokeReason   string
-	CreatedBy      string
-	CreatedAt      time.Time
-	UpdatedAt      time.Time
+	ID             string      `json:"id" yaml:"id"`
+	Status         GrantStatus `json:"status" yaml:"status"`
+	AccountID      string      `json:"account_id" yaml:"account_id"`
+	AccountType    string      `json:"account_type" yaml:"account_type"`
+	ResourceID     string      `json:"resource_id" yaml:"resource_id"`
+	Role           string      `json:"role" yaml:"role"`
+	Permissions    []string    `json:"permissions" yaml:"permissions"`
+	ExpirationDate *time.Time  `json:"expiration_date" yaml:"expiration_date"`
```

Comment:
Just wondering, why this needs to be a pointer?

Reply:
null is a possible value for permanent access case